src/storage/mvcc/txn.rs

```diff
@@ -143,7 +143,7 @@ impl<S: Snapshot> MvccTxn<S> {
 
     // If the value is short, lock key and put value.
     // If not, lock key.
-    fn put_lock(
+    fn lock_key_and_put_value_if_short(
```

Comment:
The logic is `lock_key_and_put_value_if_long`.
And I think the name is too complex.
How about just `prewrite_key`?

Reply:
Yes. I prefer `prewrite_key_value`